lib/dpl/provider/releases.rb

```diff
@@ -3,7 +3,12 @@ class Provider
     class Releases < Provider
       require 'pathname'
 
-      requires 'octokit', version: '~> 4.3.0' # later versions require Ruby 2.x
+      if RUBY_VERSION >= "2.0.0"
+        requires 'octokit', version: '~> 4.6.2'
+      else
+        requires 'octokit', version: '~> 4.3.1'
```

Comment:
Could you try `~> 4.3.0` here? https://travis-ci.org/travis-ci/dpl/jobs/185589375#L576